package.json

```
@@ -81,10 +81,10 @@
     "eslint-plugin-promise": "^6.1.1",
     "eslint-plugin-testing-library": "^5.9.1",
     "eslint-plugin-unicorn": "^45.0.0",
-    "i18next": "^22.0.6",
+    "i18next": "23.0.2",
```

Should these dependencies have carets?

```suggestion
    "i18next": "^23.0.2",
```

---

Ah good catch. Fixed.